src/main/java/org/broadinstitute/hellbender/tools/copynumber/formats/collections/DenoisedCopyRatioCollection.java

```diff
@@ -0,0 +1,51 @@
+package org.broadinstitute.hellbender.tools.copynumber.formats.collections;
+
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.copynumber.formats.records.LinearNonLocatableCopyRatio;
+import org.broadinstitute.hellbender.tools.copynumber.gcnv.GermlineCNVNamingConstants;
+import org.broadinstitute.hellbender.utils.tsv.DataLine;
+import org.broadinstitute.hellbender.utils.tsv.TableColumnCollection;
+import org.broadinstitute.hellbender.tools.copynumber.GermlineCNVCaller;
+
+import java.io.File;
+import java.util.function.BiConsumer;
+import java.util.function.Function;
+
+/**
+ * Collection of denoised copy ratios generated by {@link GermlineCNVCaller}
+ *
+ * @author Andrey Smirnov &lt;asmirnov@broadinstitute.org&gt;
+ */
+public class DenoisedCopyRatioCollection extends AbstractSampleRecordCollection<LinearNonLocatableCopyRatio> {
```

Comment:
Actually, I think you might have accidentally switched or forgotten some things in the renaming?  You should have a `NonLocatableLinearCopyRatioCollection` and a `LinearCopyRatioCollection` when the dust settles.  Make sure javadocs match if you use IntelliJ to do the renaming automatically.

Reply:
Oops changed it now!